cheroot/connections.py

```diff
@@ -152,17 +152,18 @@ def put(self, conn):
                 conn.socket.fileno(), selectors.EVENT_READ, data=conn,
             )
 
-    def _expire(self):
+    def _expire(self, threshold):
         """Expire least recently used connections.
 
-        This happens if there are either too many open connections, or if the
-        connections have been timed out.
+        Args:
+            threshold (float): Connections that have not been used
+                within this duration (in seconds), are considered
+                expired and are closed and removed.
```

Comment:
Let's use Sphinx's native param syntax:
```suggestion
        r"""Expire least recently used connections.

        :param threshold: Connections that have not been used within this \
                          duration (in seconds), are considered expired and \
                          are closed and removed.
        :type threshold: float
```